recipe/meta.yaml

```diff
@@ -94,6 +94,202 @@ outputs:
         - ibis.backends.pyspark
         - ibis.backends.sqlite
 
+
+  - name: ibis-clickhouse
```

Comment:
Thoughts on calling these `{{ name }}-backend`? It kind of sucks (the ibis pypi package is unrelated and existed before Wes created ibis), but it's consistent with `ibis-framework-core`/`ibis-framework`.

Reply:
Happy to do either -- I went with `ibis-$backend` because there are already packages called `ibis-mssql`, `ibis-omniscidb` and `ibis-bigquery`

Reply:
Ah, that makes sense. Let's keep it as you have it then.